styles/styles.css

```diff
@@ -15,35 +15,653 @@ li {
+  margin: 0;
+  list-style: none;
+  position: relative;
+}
+
+nav ul li {
+  margin: 0px;
+  display: inline-block;
+  float: left;
+  background-color: var(--primary);
+}
+
+nav ul li i {
+  margin-left: 5px;
+}
+
+nav a {
+  display: block;
+  padding: 14px 20px;
+  text-decoration: none;
+  color: var(--primary-font-color);
+  font-size: 0.9em;
+  text-transform: uppercase;
+  text-align: left;
+  font-weight: 600;
+}
+
+nav a:hover {
+  background-color: var(--gray-color);
+}
+
+nav ul ul {
+  display: none;
+  position: absolute;
+}
+
+nav ul li:hover > ul {
+  display: inherit;
+  z-index: 1;
+}
+
+nav ul ul li {
+  width: 170px;
+  float: none;
+  display: list-item;
+  position: relative;
+  z-index: 1;
+}
+
+.menu {
+  margin-top: 20px;
+}
+
+.menu-icon > .line {
+  background-color: var(--primary-font-color);
+  height: 2px;
+  display: block;
+}
+.menu-icon > .line + .line {
+  margin-top: 8px;
+}
+
+.main-toggle {
+  width: 40px;
+}
+
+.main-toggle:hover {
+  background: var(--primary);
+}
+
+#burger {
+  width: 40px;
+  height: 20px;
+  position: relative;
+  margin: 10px auto;
+  -webkit-transform: rotate(0deg);
+  -moz-transform: rotate(0deg);
+  -o-transform: rotate(0deg);
+  transform: rotate(0deg);
+  -webkit-transition: 0.5s ease-in-out;
+  -moz-transition: 0.5s ease-in-out;
+  -o-transition: 0.5s ease-in-out;
+  transition: 0.5s ease-in-out;
+  cursor: pointer;
+}
+
+#burger span {
+  display: block;
+  position: absolute;
+  height: 5px;
+  width: 100%;
+  background: var(--primary-font-color);
+  border-radius: 9px;
+  opacity: 1;
+  left: 0;
+  -webkit-transform: rotate(0deg);
+  -moz-transform: rotate(0deg);
+  -o-transform: rotate(0deg);
+  transform: rotate(0deg);
+  -webkit-transition: 0.25s ease-in-out;
+  -moz-transition: 0.25s ease-in-out;
+  -o-transition: 0.25s ease-in-out;
+  transition: 0.25s ease-in-out;
+}
+
+#burger span:nth-child(1) {
+  top: 0px;
+}
+
+#burger span:nth-child(2) {
+  top: 10px;
+}
+
+#burger span:nth-child(3) {
+  top: 20px;
+}
+
+#burger.open span:nth-child(1) {
+  top: 10px;
+  -webkit-transform: rotate(135deg);
+  -moz-transform: rotate(135deg);
+  -o-transform: rotate(135deg);
+  transform: rotate(135deg);
+}
+
+#burger.open span:nth-child(2) {
+  opacity: 0;
+  left: -60px;
+}
+
+#burger.open span:nth-child(3) {
+  top: 10px;
+  -webkit-transform: rotate(-135deg);
+  -moz-transform: rotate(-135deg);
+  -o-transform: rotate(-135deg);
+  transform: rotate(-135deg);
+}
+/* ========================== End Navigation ========================== */
+
+/* ========================== Hero Block ========================== */
+#hero-block {
+  display: grid;
+  background-image: url("../assets/images/hero.png");
+  height: 400px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+#hero-text .block-title {
+  margin-bottom: 25px;
+}
+
+#hero-text {
+  text-align: left;
+  color: white;
+}
+
+#hero-text div {
+  position: absolute;
```

Comment:
Recommended using a static position of text above the image. Otherwise, text could exceed the height of the wrapper.